public/css/common.less

```diff
@@ -328,3 +328,36 @@ div.show-more {
 .servicegropup-list > col.object-statistics-graph {
   padding-right: .25em;
 }
+
+ul.item-list li:last-child .comment-wrapper {
+  .comment-popup {
+    top: -7em;
+  }
+
+  .comment-popup:before {
+    bottom: -0.86em;
+    transform: rotate(225deg);
+  }
+}
+
+#layout {
+  &.twocols {
+    .comment-popup {
+      width: 35em;
+    }
+
+    &.poor-layout,
+    &.compact-layout {
+      .comment-popup {
+        width: 25em;
+      }
+    }
+  }
+
+  &.poor-layout,
+  &.minimal-layout {
+    .comment-popup {
+      width: 25em;
+    }
+  }
+}
```

Comment:
That's also related to the popup only

Reply:
It is only displayed with committed changes. I don't have any local changes. Maybe it only appears on my site and it works fine elsewhere?

Reply:
What do you mean? That's the current pushed state, isn't it?

Reply:
Yes. The current pushed state.